Update python modules

RobLoach opened the pull request:
Getting the following error:

```
Namespace GnomeDesktop not available
```

Fixes #18

RobLoach committed:
Update python modules

LeandroStanger commented:
```
erro: Unable to load summary from remote lutris: Remote listing for lutris not available; server has no summary file. Check the URL passed to remote-add was valid.
```

LeandroStanger commented:
```
Error: Failed to install org.lutris.Lutris/x86_64/master: Failed to parse /sysroot/home/leandro/.local/share/flatpak/appstream/lutris/x86_64/active/appstream.xml.gz file: Error getting information for file “/sysroot/home/leandro/.local/share/flatpak/appstream/lutris/x86_64/active/appstream.xml.gz”: File or directory not found
```

AsciiWolf commented on org.lutris.Lutris.yaml:
```diff
@@ -4,7 +4,7 @@ runtime: org.gnome.Platform
 command: lutris
-rename-desktop-file: lutris.desktop
+rename-desktop-file: net.lutris.Lutris.desktop
 rename-appdata-file: lutris.appdata.xml
```
This should be called `net.lutris.Lutris.appdata.xml`.

RobLoach committed:
Fix net.lutris.Lutris.appdata.xml

Thanks @AsciiWolf

AsciiWolf commented:
@RobLoach Why is the Flatpak ID and this repo name `org.lutris.Lutris`? The correct name is, afaik, `net.lutris.Lutris`.

RobLoach commented:
Likely legacy. We can change it.

RobLoach committed:
Rename to net.lutris.Lutris

AsciiWolf commented on net.lutris.Lutris.yaml:
```diff
@@ -1,11 +1,10 @@
 command: lutris
-rename-desktop-file: lutris.desktop
-rename-appdata-file: lutris.appdata.xml
+rename-appdata-file: net.lutris.Lutris.appdata.xml
```
The `rename-appdata-file` line should not be needed anymore as well. :-)

AsciiWolf replied:
Anyway, there is currently a small issue with Lutris AppData file having bad desktop file ID. See https://github.com/lutris/lutris/pull/1808.

AsciiWolf replied:
Is the `rename-appdata-file` line somehow useful anymore?

RobLoach replied:
Unsure! Haven't tried to build it since I pushed that forwards. We may not need it.

Biggest blocker is GnomeDesktop namespace not being there when running.

LeandroStanger replied:
GnomeDesktop is included in runtime https://gitlab.gnome.org/GNOME/gnome-build-meta/blob/master/elements/core/gnome-desktop.bst

alatiera replied:
Its not, what's in the runtime is defined in this [file][1]. `gnome-desktop` is a highly unstable component (on purpose) that [won't be part][2] of the runtime. Please bundle it, all the other apps do so as well.

[1]: https://gitlab.gnome.org/GNOME/gnome-build-meta/blob/9fa4f664f0c562bd11432be8eee8733773f10c6c/elements/sdk-platform.bst
[2]: https://gitlab.gnome.org/GNOME/gnome-build-meta/issues/125#note_447760

LeandroStanger committed:
Stop bundling Python 

Python is included in runtime https://gitlab.com/freedesktop-sdk/freedesktop-sdk/blob/18.08/elements/base/python3.bst

LeandroStanger committed:
Update runtime to 3.32

LeandroStanger committed:
Update to v0.5.1

RobLoach commented:
Still getting the GnomeDesktop error...

```
$ flatpak run net.lutris.Lutris
Traceback (most recent call last):
  File "/app/bin/lutris", line 31, in <module>
    from lutris.gui.application import Application
  File "/app/lib/python3.7/site-packages/lutris/gui/application.py", line 30, in <module>
    gi.require_version("GnomeDesktop", "3.0")
  File "/usr/lib/python3.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace GnomeDesktop not available
```

AsciiWolf commented:
Is it possible to bundle the gnome-desktop library?

LeandroStanger commented:
https://github.com/flathub/flathub/pull/919
